src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs

```diff
@@ -272,5 +278,47 @@ public void Dispose()
             clientData.pSignerParams->pSignCallBack = signInfo;
 
         }
-    }
+
+        /// <summary>
+        /// It is required to call initialize before using the AzureSign to ensure dll's are loaded correctly.
+        /// The initialize should happen as early as possible in the host process.
+        /// </summary>
+        /// <param name="assemblyPath">Override the default location for the assembly. Necessary when running on Azure App Services</param>
+        public static void Initialize(string assemblyPath = null)
+        {
+            if (ManifestLocation != null)
+                return; // already initialized
+
+
+            var is64bit = IntPtr.Size == 8;
```

Comment:
Can we use [`Environment.Is64BitProcess`](https://docs.microsoft.com/en-us/dotnet/api/system.environment.is64bitprocess?view=netcore-2.1)? There's nothing "wrong" with this code but it seems to express the intent better to a reader.

Reply:
I've never been sure if one of them lies?

Reply:
It does the exact same thing as your code.

https://github.com/dotnet/corefx/blob/28e68e5426332a0f45c5fa5b2c221f74d3bed032/src/System.Runtime.Extensions/src/System/Environment.cs#L141